site-src/concepts/api-overview.md

```diff
@@ -1,3 +1,31 @@
 # API Overview
 
-TODO
\ No newline at end of file
+## Bakcground
+The Gateway API Inference Extension project is an extension of the Kubernetes Gateway API for serving Generative AI models on Kubernetes. Gateway API Inference Extension facilitates standardization of APIs for Kubernetes cluster operators and developers running generative AI inference, while allowing flexibility for underlying gateway implementations (such as Envoy Proxy) to iterate on mechanisms for optimized serving of models. 
+
+<img src="/images/inference-overview.png" alt="Overview of API integration" class="center" width="700" />
+
+
+## Key Features 
```

Comment:
Should we put this in the introduction https://gateway-api-inference-extension.sigs.k8s.io/

and also remove the API Resources section from that page since we now have them here (see https://gateway-api-inference-extension.sigs.k8s.io/#api-resources)